performance_test/src/experiment_execution/analyze_runner.cpp

```diff
@@ -20,14 +19,19 @@
 #include <string>
 #include <vector>
 
+#ifdef PERFORMANCE_TEST_ODB_FOR_SQL_ENABLED
+  #include <exception>
+  #include <memory>
+#endif
```

Comment:
I think we can simplify this.  There is really no need to put the `#include <exception>` and `#include <memory>` within the `#ifdef`; we know they are always going to be available.  So:

```suggestion
#include <exception>
#include <memory>
```

(and then move them into place in the C++ header list above)